portfolio/src/main/webapp/js/script.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Switches Light Mode of Page
+ */
+function switchMode() {
```

Comment:
I'm impressed you found a way to get day and night mode working! This is a pretty cool feature.

I do want to take the opportunity to tell you a bit about an alternate solution though, and the reasons behind it. Note that you don't need to make this change in this PR - this is just to try to let you see alternate solutions.

CSS works by "cascading" rules, which means that you can have a global switch in the html and keep everything in one stylesheet. Let's take the example of `.bg-mode` for the background color as an example. Here's what you currently have:
```
// day mode
.bg-mode {
  background-color: #ffffff;
}

// night mode
.bg-mode {
  background-color: #2f353a;
}

// html (simplified)
<body>
  <div class="bg-mode">
    ...
  </div>
</body>
```

One other way to implement this is the following:
```
// one shared css file (styles.css)
.night .bg-mode {
  background-color: #2f353a;
}
.day .bg-mode {
  background-color: #fff;
}

// html
<body class="day">
  <div class="bg-mode">
    ...
  </div>
</body>

// javascript
function switchMode() {
  var bodyElement = document.getElementById("page-top");
  if (bodyElement.className == "day") {
    bodyElement.className = "night";
  } else {
    bodyElement.className = "day";
  }
}
```

The main advantage of doing it this way is that you don't need to load two stylesheets. When you change the href value, the browser needs to load a new resource, which can take time especially on mobile browsers. In the meantime, the user might see some weird styles because you "took away" the original stylesheet.